test-case/check-pause-release-suspend-resume.sh

```diff
@@ -169,7 +169,17 @@ expect {
         if { \$i > $repeat_count } { exit 0 }
         exp_continue
     }
+    "*:*\n" {
+        puts "\rexpect ignore dump-hw-params"
+        after 10
+        exp_continue
+    }
+    "^?*\n" {
+        puts "\rError: unexpected output detected, exit 1"
```

Comment:
Please print what was unexpected, should be something like `expect_out(0,string)` (untested)

Reply:
Updated, I couldn't see meaningful output from my test but it seems useful when it hit the case